lesson5/homework_05.py

```diff
@@ -0,0 +1,56 @@
+# task 1. Знайдіть всі унікальні елементи в списку small_list
+small_list = [3, 1, 4, 5, 2, 5, 3]
+unique_elements = [x for x in small_list if small_list.count(x) == 1]
+print(unique_elements)
+
+# task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
+small_list = [3, 1, 4, 5, 2, 5, 3]
+average = sum(small_list) / len(small_list)
+print("average is " , average)
+
+# task 3. Перевірте, чи є в списку big_list дублікати
+big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
+set_big_list = set(big_list)
+if big_list == set_big_list :
```

Comment:
```suggestion
if big_list == set_big_list :
```
так ніколи не буде працювати, бо сет - невпорядкований, і не можна брати напряму == з лістом